### ISSUE TITLE
comment language

### ISSUE THREAD

**Issue by cnmicha** [Herder-IT-Solutions/HerderBib#137]

**Comment by JanMich** [comment]
why? this should be understandable by everyone!

**Comment by PSchlesselmann** [comment]
Well, everyone should be able to understand english.

**Comment by bvorak** [comment]
I feel like it would make more sense to have consistent (English) code comments than to lead discussions like this one in English...:P 
But if you really feel uncomfortable about English comments leave them German for now I'd say...translating them isn't necessarily the most important thing at the moment. 
But we should ensure that the content and style is somewhat consistent across units.

**Comment by PSchlesselmann** [comment]
Yeah, in case you are not familiar enough with this language: Just keep your comments in german.
We will translate them later.

**Comment by frcroth** [comment]
The GUI doesn't even have comments...

**Comment by ghost** [comment]
In my opinion comments in english are not useful because it is for your school that will alway be german and why should we run danger that there are misconceptions because of the language.
Ok it looks professional but for example a software developer would never write instructions in english when he has an german customer.
We should guarentee that everyone understand our instructions and we could not if we write our comments in english. Also we should estimate the time users have to spend by look up words they don`t know.

**Comment by bvorak** [comment]
And we also need to remember that right now we are mainly coding for Kraxeleddi ^^

**Comment by PSchlesselmann** [comment]
Well how about just asking him about his preference?
Then we can be sure and this discussion finally becomes unnecessary.

**Comment by PSchlesselmann** [comment]
I asked him: He prefers german.

**Comment by bvorak** [comment]
So we should probably translate all of our commments to German then I guess...maybe it's not a must until Friday though if we have a good Documentation!
